src/main/java/de/uni_tuebingen/ub/nppm/model/SelektionTitelKritik.java

```diff
@@ -0,0 +1,31 @@
+package de.uni_tuebingen.ub.nppm.model;
+
+import javax.persistence.*;
+import java.util.*;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+@Entity
+@Table(name = "selektion_titelkritik")
+@Cacheable
+@org.hibernate.annotations.Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+public class SelektionTitelKritik extends SelektionBezeichnung{
+
+     @ManyToMany(mappedBy = "titelKritiken")
+    private Set<Einzelbeleg> einzelbeleg = new HashSet<>();
+
+    public Set<Einzelbeleg> getEinzelbeleg() {
+        return einzelbeleg;
+    }
+
+    public void setEinzelbeleg(Set<Einzelbeleg> einzelbeleg) {
```

Comment:
Here as well, i dont think we ever need this setter and should switch to add/remove single ones to the list.

Reply:
done